### DISCUSSION TITLE
1.1.0-preview3 "Unknown attachable member 'BackdropMaterial.ApplyToRootOrPageBackground' on element 'Page'"

### DISCUSSION THREAD

**Discussion by otomad** [microsoft/WindowsAppSDK#2485] [Q&A]
I still can't use the mica backdrop in 1.1.0-preview3.

"Unknown attachable member 'BackdropMaterial.ApplyToRootOrPageBackground' on element 'Page'"

![image](https://user-images.githubusercontent.com/56647156/167243212-a84b6530-a10b-4b64-8e16-edb97a3b94fb.png)
![image](https://user-images.githubusercontent.com/56647156/167243224-3ed2441b-7409-45c1-a4ac-473b7f6cc9bb.png)
![image](https://user-images.githubusercontent.com/56647156/167243259-ad75da68-0658-49bd-b5e9-3c282a6f4a54.png)

**Comment by ghost1372** [comment]
There isnt attached property, you need to write some codes, you can use my project and SystemBackdropsHelper

https://github.com/ghost1372/SettingsUI
